.github/workflows/mypy.yaml

```diff
@@ -29,7 +29,8 @@ jobs:
           cache-environment: true
           cache-downloads: true
           create-args: >-
-            python=3.9
+            python=3.10
+            rdkit=2023.09.5
```

Comment:
I've never been able to work that one out.

Reply:
@IAlibay can you elaborate? Work what out here?

Reply:
For some reason or another mypy has been complaining about a certain error with rdkit 2024+. Something about self being defined twice for a certain object.

Our approach has just been to pin to an older version of rdkit, but honestly we should try to fix it properly.